nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -66,14 +65,17 @@
 except ImportError:
     amp = None
 
-__all__ = ["ImagenetExperiment"]
+__all__ = [
+    "SupervisedExperiment",
+    "ImagenetExperiment",
+]
 
 
 # Improves performance when using fixed size images (224) and CNN
 cudnn.benchmark = True
 
 
-class ImagenetExperiment:
+class SupervisedExperiment:
```

Comment:
Is `SupervisedExperiment` specific for `resnet50 v1.5` on `Imagenet` or generic for any model on any dataset?
If it is generic then it should not be in the `imagenet_experiment.py` file

Reply:
It is generic. I've yet to do the renaming of all the files, however (including the imagenet directory to yael of whatever else we decide). If it's alright, the renaming will happen soon in a future PR.

Reply:
If you feel strongly about this, I can also do all the renaming now, ofc.

Reply:
It is ok to rename the file in the future but you should at least update the docstring

Reply:
Can do.